### ISSUE TITLE
How to divide the data set

### ISSUE THREAD

**Issue by qqiuqq** [MeimeiZhang-data/MQVQA#4]
Can you provide the results of the training set, test set and validation set of RSIVQA in your paper?
